iot_input_oca/models/iot_device.py

```diff
@@ -10,6 +10,7 @@ class IotDevice(models.Model):
 
     input_ids = fields.One2many("iot.device.input", inverse_name="device_id")
     input_count = fields.Integer(compute="_compute_input_count")
+    last_contact_date = fields.Datetime(readonly=True)
```

Comment:
Shouldn't this be on main module? Both input or output should write to this field.